pkg/capacity/manager_test.go

```diff
@@ -166,8 +166,8 @@ func TestGetOptimalTargetCluster(t *testing.T) {
 	t.Run("with two clusters but only the second one has enough capacity - using the default values", func(t *testing.T) {
 		// given
 		toolchainConfig := commonconfig.NewToolchainConfigObjWithReset(t,
-			testconfig.AutomaticApproval().
-				MaxNumberOfUsers(2000).
+			testconfig.CapacityThresholds().
+				MaxNumberOfSpaces(testconfig.PerMemberCluster("member1", 700), testconfig.PerMemberCluster("member2", 1000)).
```

Comment:
we should keep it empty because it's testing the memory limit
```suggestion
				MaxNumberOfSpaces().
```